components/brave_ads/browser/ads_service.h

```diff
@@ -157,6 +160,10 @@ class AdsService : public KeyedService {
   void RemoveObserver(
       AdsServiceObserver* observer);
 
+  // static
+  static void RegisterProfilePrefs(
```

Comment:
if you make this non-static override it will automatically get called

Reply:
Changes, thanks